example/mutatingwebhook.go

```diff
@@ -62,19 +61,15 @@ func (a *podAnnotator) mutatePodsFn(ctx context.Context, pod *corev1.Pod) error
 	return nil
 }
 
-// podValidator implements inject.Client.
 // A client will be automatically injected.
-var _ inject.Client = &podValidator{}
 
 // InjectClient injects the client.
 func (v *podAnnotator) InjectClient(c client.Client) error {
```

Comment:
If I understand correctly, I can get rid of this method by just adding a tag `inject` on client field in `podAnnotator` ?

Reply:
Got it. We should document it at an appropriate place, because users will have the same question.